formfactory/templates/formfactory/form_detail.html

```diff
@@ -1,11 +1,12 @@
 {% extends "base.html" %}
+{% load formfactory_tags %}
 
 {% block content %}
-    <form class="Form" method="POST" action="." enctype="multipart/form-data">
-        {% if view.form_object.enable_csrf %}
-            {% csrf_token %}
-        {% endif %}
-        {{ form.as_p }}
-        <input type="submit" value="{{ view.form_object.submit_button_text }}" />
-    </form>
+    {% comment %}
+        We by default use the render form tag here. This means we don't need
+        duplicate js for ajax posting on base forms. Also allowing users to use
+        the existing detail form and only needing to override the inner
+        inclusion detail template.
+    {% endcomment %}
+    {% render_form object %}
```

Comment:
I don't see why this should be the default here. It looks like this will call the view twice.
Scenario:
You have a form with `slug = "login-fom"`.
- A user accesses `/formfactory/login-form/`.
- by default, that uses `formfactory/form_detail.html`, which in turn calls `render_form`.
- `render_form` resolves the view.

I understand the need for for what you're suggesting, but I don't think `render_form` should be in this default template. 

Considering that you already do have
```
template = self.kwargs.get("template", None)
``` 
in `get_template_names`, then perhaps you could pass the name of a different template to acheve the desired functionality without affecting this template.